### PR TITLE
Update devdocs.css

### PR DESCRIPTION
**Pull request by aryandesssigner**
## Purpose of this pull request

This pull request for fixing space issue between button of page-survey-questions

## Affected DevDocs pages

All devdoc page where survey button is present

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@aryandesssigner thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: src/common/css/devdocs.css [hunk 1]
```diff
@@ -22,3 +22,8 @@
 .container > .feature > ul > li > img.img-icon {
   margin-top: -4px;
 }
+
+/* page-survey-questions button */
+.page-survey-buttons .spectrum-Button--secondary:first-child {
+    margin-right: 10px;
```
Comment (BarnyShergold) [inline src/common/css/devdocs.css:28]:
```suggestion
  margin-right: 10px;
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
There are some issues causing linting errors in other areas of the file that will need fixing -
Line 11 - insert space between dt and {
Line 12 - insert space after : 
Line 19 - wants a leading zero but not sure why!

You need to sign the Adobe CLA and ensure the Linting checks pass.

**Comment by BarnyShergold** [comment]
You still need to correct the linting errors

**Comment by BarnyShergold** [comment]
There seems to be a whitespace the end of EVERY new line in devdocs.css

**Comment by bdenham** [comment]
We really don't want to be adding overriding styles outside of the https://github.com/magento-devdocs/devdocs-theme repo. I will soon move or remove all stylesheets that are currently a part of `devdocs/common/css` to the devdocs theme repo. Spreading styles between the theme and the devdocs project results in a maintenance nightmare.

Does this make sense to everyone? @belbiy?

**Comment by dobooth** [comment]
just running a test here....

**Comment by dobooth** [comment]
@magento import pr to magento-commerce/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request closed without merging**